phpcs-ruleset.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<!-- PHP_CodeSniffer standard -->
+<!-- See https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
+<ruleset name="Invoicing">
+    <description>DrupalDriver coding standard</description>
+
+    <rule ref="./vendor/drupal/coder/coder_sniffer/Drupal">
+    </rule>
+
+    <!-- Core drivers need to set a server's remote address. -->
+    <rule ref="Drupal.Semantics.RemoteAddress.RemoteAddress">
+        <exclude-pattern>src/Drupal/Driver/Cores/*.php</exclude-pattern>
+    </rule>
```

Comment:
Does this rule exclude all core drivers? And if so, why? If they do need to be excluded, can we clarify the reason in the comment (I don't understand the current comment about remote server address).